### PR TITLE
Move Date Range selection to Query Page

### PR DESCRIPTION
**Pull request by BradleySappington**
Remove the Date Range Archive page and its functionality completely.
Implement date range picker on main query page so it can be utilized with all other query filters.

### CODE REVIEW

**Comment by BradleySappington** [comment]
> Hello @BradleySappington, Thank you for updating !
> 
> * In the file [`jwql/utils/constants.py`](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/utils/constants.py):
> 
> > [Line 1002:25](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/utils/constants.py#L1002): [W504](https://duckduckgo.com/?q=pep8%20W504) line break after binary operator
> > [Line 1003:26](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/utils/constants.py#L1003): [W504](https://duckduckgo.com/?q=pep8%20W504) line break after binary operator
> > [Line 1004:30](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/utils/constants.py#L1004): [W504](https://duckduckgo.com/?q=pep8%20W504) line break after binary operator
> > [Line 1005:38](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/utils/constants.py#L1005): [W504](https://duckduckgo.com/?q=pep8%20W504) line break after binary operator
> > [Line 1006:29](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/utils/constants.py#L1006): [W504](https://duckduckgo.com/?q=pep8%20W504) line break after binary operator
> > [Line 1007:23](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/utils/constants.py#L1007): [W504](https://duckduckgo.com/?q=pep8%20W504) line break after binary operator
> 
> * In the file [`jwql/website/apps/jwql/data_containers.py`](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/website/apps/jwql/data_containers.py):
> 
> > [Line 410:5](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/website/apps/jwql/data_containers.py#L410): [E129](https://duckduckgo.com/?q=pep8%20E129) visually indented line with same indent as next logical line
> > [Line 410:17](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/website/apps/jwql/data_containers.py#L410): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> > [Line 411:21](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/website/apps/jwql/data_containers.py#L411): [E117](https://duckduckgo.com/?q=pep8%20E117) over-indented
> > [Line 420:56](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/website/apps/jwql/data_containers.py#L420): [W504](https://duckduckgo.com/?q=pep8%20W504) line break after binary operator
> > [Line 421:5](https://github.com/spacetelescope/jwql/blob/ea4ae52c2a4a1b80fb3834972455f2eb22dd0d0b/jwql/website/apps/jwql/data_containers.py#L421): [E129](https://duckduckgo.com/?q=pep8%20E129) visually indented line with same indent as next logical line
> 
> ##### Comment last updated at 2023-11-08 17:33:50 UTC

PEP8 is enforcing 2 conflicting requirements w503 and w504

**Comment by BradleySappington** [comment]
@mfixstsci @bhilbert4 - ready for review

**Comment by BradleySappington** [comment]
https://jira.stsci.edu/browse/JWQL-142

File: jwql/website/apps/jwql/templates/base.html [hunk 1]
```diff
@@ -27,6 +27,11 @@
 		<script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-{{ bokeh_version }}.min.js"></script>
 		<script src="{{ static('js/jwql.js') }}?{% now 'U' %}"></script>
 
+		<!-- Needed for Daterange on Query Page -->
+		<script type="text/javascript" src="https://cdn.jsdelivr.net/momentjs/latest/moment.min.js"></script>
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/templates/base.html:31]: Nice

**Review by mfixstsci — APPROVED** [review]
@BradleySappington Thanks for your work on this! The changes on the query page look great and I think it cleans up the archive page at the same time.

**Pull request merged**